x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/MlDailyMaintenanceService.java

```diff
@@ -70,9 +69,14 @@ public MlDailyMaintenanceService(ClusterName clusterName, ThreadPool threadPool,
     private static TimeValue delayToNextTime(ClusterName clusterName) {
         Random random = new Random(clusterName.hashCode());
         int minutesOffset = random.ints(0, MAX_TIME_OFFSET_MINUTES).findFirst().getAsInt();
-        DateTime now = DateTime.now(ISOChronology.getInstance());
-        DateTime next = now.plusDays(1).withTimeAtStartOfDay().plusMinutes(30).plusMinutes(minutesOffset);
-        return TimeValue.timeValueMillis(next.getMillis() - now.getMillis());
+
+        ZonedDateTime now = ZonedDateTime.now();
```

Comment:
UTC tz

Reply:
here again originally no time zone was specified. 
`DateTime.now(ISOChronology.getInstance());` is an equivalent of `DateTime.now()` 
ISOChronology is a default and system default time zone will be defaulted too.

Reply:
@droberts195 would you be able to comment on these? Do you think it is safe if we refactor this to use UTC timezones, given it wasn't previously specified?